apps/meteor/app/api/server/v1/im.ts

```diff
@@ -400,10 +400,9 @@ API.v1.addRoute(
 				...parseIds(starredIds, 'starred._id'),
 				...(pinned && pinned.toLowerCase() === 'true' ? { pinned: true } : {}),
 			};
-			const sortObj = { ts: sort?.ts ?? -1 };
 
 			const { cursor, totalCount } = Messages.findPaginated(ourQuery, {
-				sort: sortObj,
+				sort: sort,
```

Comment:
I believe it would be good to have a default (ts) in case nothing is provided.

Reply:
Also, can we add a test case to cover this new behavior?

Reply:
Yes, it's a great idea to add a test to the suite to ensure everything works as expected. I have implemented that test, along with the default ts (timestamp) sorting behavior you mentioned. I believe your point about the default ts value is valid, and it makes the implementation more robust.